sse2neon.h

```diff
@@ -2315,6 +2315,15 @@ FORCE_INLINE __m128 _mm_add_ps(__m128 a, __m128 b)
         vaddq_f32(vreinterpretq_f32_m128(a), vreinterpretq_f32_m128(b)));
 }
 
+#if defined(__aarch64__)
+FORCE_INLINE __m128d _mm_add_pd(__m128d a, __m128d b)
+{
+    return (__m128d) vreinterpretq_m128_f32(
+        vaddq_f64(vreinterpretq_f64_m128((float32x4_t) a),
+                  vreinterpretq_f64_m128((float32x4_t) b)));
```

Comment:
A64:
```c
    return vreinterpretq_m128d_f64(
        vaddq_f64(vreinterpretq_f64_m128d(a), vreinterpretq_f64_m128d(b)));
```

The `vreinterpretq_f64_m128` and `vreinterpretq_m128_f64` should be implemented.
However, it would not be hard.

A32:
```c
    double *aptr = (double *) &a;
    double *bptr = (double *) &b;
    double c[2];
    c[0] = aptr[0] + bptr[0];
    c[1] = aptr[1] + bptr[1];
    return vld1q_f32((float32_t *)c);
```